### ISSUE TITLE
Abandon coffeescript in favor of ES6

### ISSUE THREAD

**Issue by VanTanev** [leocaseiro/angular-chosen#255]
I propose rewriting the module in ES6 for version 2.0
Honestly, dealing with coffee script is a bigger distraction than it's worth, imo.

**Comment by leocaseiro** [comment]
I totally agree. IMO I never really liked coffeescript. It was more just a challenge.

Happy to use es6 or TS. But TS will be confused with Angular 2-6

**Comment by andresgardiol** [comment]
It would be great to be able to use chosen with es6 modules.
Will you rewrite it by chance?

**Comment by VanTanev** [comment]
@andresgardiol `angular-chosen` is separate from [chosen-js](https://github.com/harvesthq/chosen). Even if we rewrote angular-chosen to be compatible with what you mean to be (I assume) browser es6 modules, `chosen-js` itself is not, and we cannot rewrite it.

However, even the current version of both `angular-chosen` and `chosen-js` are compatible with es6 imports, as handled by WebPack.

**Comment by andresgardiol** [comment]
Thanks for your answer. Can you tell me how you import angular-chosen module with es6? I had some problems to do so.

**Comment by VanTanev** [comment]
This really isn't the place, but 1 short example
```js
import 'jquery'
import 'chosen-js'
import 'angular-chosen-localytics'
import angular from 'angular'

angular.module('app', ['localytics.directives'])
```
